charts/zaakbrug/templates/configmap.yaml

```diff
@@ -43,6 +43,21 @@ metadata:
   labels:
     {{- include "zaakbrug.labels" . | nindent 4 }}
 data:
+  {{- with .Values.zaakbrug.endpoints.soap }}
+  zgw.baseurl: {{ .baseurl }}
```

Comment:
Deze is dubbel en ik zie ook geen .Values.zaakbrug.endpoints.soap.baseurl

Reply:
Goed gespot! Copy/paste.